components/02-elements/rating/_rating.scss

```diff
@@ -1,78 +1,105 @@
-$rating-font-size: $font-size-large;
-$rating-active   : $orange;
+$rating__size: 96px;
 
 .rating {
-    unicode-bidi: bidi-override;
-    direction: rtl;
-    font-size: $rating-font-size;
     cursor: pointer;
-    text-align: left;
+    width: $rating__size;
 
-    span {
-        display: inline-block;
+    &--rate {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        direction: rtl;
+    }
+    &__rate-item {
+        flex: 1 0 ($rating__size/5);
+        overflow: hidden;
         position: relative;
-
         &:hover {
-            &:before {
-                content: "\2605";
-                position: absolute;
-                color: $rating-active;
-            }
 
-            ~ span {
-                &:before {
-                    content: "\2605";
-                    position: absolute;
-                    color: $rating-active;
-                }
+            ~ .rating__rate-item span:before {
+                display: block;
             }
         }
     }
-
-    &__summary {
+    &__star {
+        height: ($rating__size/5);
         position: relative;
-        display: inline-block;
-        width: 92px;
-        letter-spacing: 1px;
-        height: $rating-font-size;
-        font-size: $rating-font-size;
-        line-height: $rating-font-size;
-        vertical-align: middle;
-
-        &:before {
-            content: '\2605' '\2605' '\2605' '\2605' '\2605';
-            position: absolute;
-            left: 0;
-            width: 100%;
-            height: 100%;
+        width: $rating__size;
+        &--rate {
+            width: ($rating__size/5)
+        }
+        &:before { // inactive
+            background-image: url("data:image/svg+xml,%3Csvg viewBox='0 0 24 19' xmlns='http://www.w3.org/2000/svg' fill-rule='evenodd' clip-rule='evenodd' stroke-linejoin='round' stroke-miterlimit='1.414'%3E%3Cpath fill='none' d='M-2-2h24v24H-2z'/%3E%3Cpath d='M20 7.24l-7.19-.62L10 0 7.19 6.63 0 7.24l5.46 4.73L3.82 19 10 15.27 16.18 19l-1.63-7.03L20 7.24zM10 13.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L10 4.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L10 13.4z' fill='%23c2c1df'/%3E%3C/svg%3E ");
```

Comment:
we can add files like that inside css that shouldn't be put in the separate sprite readable for css background? I'm not saying it's a bug, just curious :)

Reply:
For this use case I think it's easier for update and maintance just to replace svg here as it's in readme. However if you or anyone can create working solution with your proposition then we could add it of course